iOS/DuckDuckGo/AutofillLoginListViewController.swift

```diff
@@ -1066,6 +963,9 @@ extension AutofillLoginSettingsListViewController: AutofillLoginDetailsViewContr
             Pixel.fire(pixel: .autofillManagementSaveLogin)
         } else {
             Pixel.fire(pixel: .autofillManagementUpdateLogin)
+            if viewModel.isSearching, let query = searchController.searchBar.text {
```

Comment:
This is a bug fix I worked in for [Autofill Bug: Passwords no longer filtered by search query after editing an item (iOS)](https://app.asana.com/1/137249556945/project/1203822806345703/task/1209495529591473)